it_hardware/data/ir_ui_view.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <record id="helpdesk_ticket_form_customization_it_hardware" model="ir.ui.view">
+        <field name="name">helpdesk.ticket.form.customization.it.hardware</field>
+        <field name="model">helpdesk.ticket</field>
+        <field name="type">form</field>
+        <field name="inherit_id" ref="helpdesk_stock.helpdesk_ticket_view_form_inherit_helpdesk_stock"/>
+        <field name="mode">extension</field>
+        <field name="priority">800</field>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='email_cc']" position="after">
+                <field name="product_id"/>
+                <field name="lot_id"/>
+                <field name="x_warranty_date" invisible="not lot_id"/>
+            </xpath>
+        </field>
+    </record>
+    <record id="stock_pr_lot_form_customization_it_hardware" model="ir.ui.view">
+        <field name="name">stock.production.lot.form.customization.it.hardware</field>
+        <field name="model">stock.lot</field>
+        <field name="type">form</field>
+        <field name="inherit_id" ref="stock.view_production_lot_form"/>
+        <field name="mode">extension</field>
+        <field name="priority">160</field>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='removal_date']" position="after">
+                <field name="x_warranty_date"/>
+            </xpath>
+        </field>
+    </record>
+    <record id="stock_pr_lot_tree_customization_it_hardware" model="ir.ui.view">
+        <field name="name">stock.production.lot.tree.customization.it.hardware</field>
+        <field name="model">stock.lot</field>
+        <field name="type">tree</field>
+        <field name="inherit_id" ref="stock.view_production_lot_tree"/>
+        <field name="mode">extension</field>
+        <field name="priority">160</field>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='product_id']" position="after">
+                <field name="x_warranty_date"/>
+            </xpath>
+        </field>
+    </record>
+    <record id="product_form_view_customization_it_hardware" model="ir.ui.view">
+        <field name="name">product.template.product.form.customization.it.hardware</field>
+        <field name="model">product.template</field>
+        <field name="type">form</field>
+        <field name="inherit_id" ref="product.product_template_only_form_view"/>
+        <field name="mode">extension</field>
+        <field name="priority">160</field>
+        <field name="arch" type="xml">
+            <data>
+                <xpath expr="//group[@name='inventory']/group[@name='expiry_and_lots']/div[4]" position="after">
```

Comment:
Could you shorten the xpath? Wouldn't this work?
```suggestion
                <field name="alert_time" position="after">
```

Reply:
![image](https://github.com/user-attachments/assets/c01b3654-f7d4-4ec7-b7d3-c697368f6185)
It will look like this

Reply:
Isn't it because of the oe_inline?

Reply:
No its not because of oe-inline